docs/_data/bearer_scan.yaml

```
@@ -37,6 +37,10 @@ options:
       default_value: '[]'
       usage: |
         Specify the comma-separated fingerprints of the findings you would like to exclude from the report.
+    - name: exit-code
+      default_value: "-1"
+      usage: |
+        Forces the exit code to be 0 (success) despite any findings from the scan
```

A suggestion to re-word this a bit, since now we can give any exit code we want

```suggestion
        Use this to force a given exit code for the scan command. Set this to 0 (success) to always return a success exit code despite any findings from the scan. 
```